core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -2415,6 +2438,21 @@ components:
           type: string
           description: |
             The hex-encoded raw bytes of the mutadata data of the NF.
+    ValidatorSubstate:
+      allOf:
+        - $ref: '#/components/schemas/SubstateBase'
```

Comment:
We have changed how we do disciminators in Core API after a PR last week - and got rid of the "Base" classes - this should instead reference Substate here